presto-main/src/test/java/com/facebook/presto/failureDetector/TestHeartbeatFailureDetector.java

```diff
@@ -88,6 +93,34 @@ public void configure(Binder binder)
         assertTrue(detector.getFailed().isEmpty());
     }
 
+    @Test
```

Comment:
When I said deserialize I was thinking of getting a `Stats` instance back instead of getting a `JsonNode` (which will fail due to reasons I mentioned above):

```
Stats deserialized = JsonCodec.jsonCodec(Stats.class).fromJson(serializedStats);
//do assertions on deserialized
```